kolibri/core/auth/management/commands/flushsyncsessions.py

```diff
@@ -18,16 +21,36 @@ class Command(AsyncCommand):
     help = "This command initiates the deletion process for temporary sync session buffers."
 
     def add_arguments(self, parser):
+        default_date = timezone.now() - timedelta(days=1)
         parser.add_argument("--noninteractive", action="store_true")
+        parser.add_argument(
+            "--before-date",
```

Comment:
When I enter `kolibri manage flushsyncsessions --before-date 2020-07-23`, I got a warning below
```
kolibri/dist/django/db/models/fields/__init__.py:1451: RuntimeWarning: DateTimeField SyncSession.last_activity_timestamp received a naive datetime (2020-07-24 00:00:00) while time zone support is active.
  RuntimeWarning)
```
Should I enter the date in another format? Thank you!

Reply:
Yeah it needs the timezone, so from the confirmation message you can get it. Mine is actually just zero, so try 
`2020-07-23T+00`

Reply:
thank you! that works! Do you think if it would be helpful to add this as a simple example in the argument description or comment?